MinPlatformPkg/Test/Library/TestPointLib/StandaloneMmTestPointLib.inf

```diff
@@ -0,0 +1,47 @@
+## @file
+# SMM instance of TestPoint Library.
+#
+# Copyright (c) 2017, Intel Corporation. All rights reserved.<BR>
```

Comment:
nit: Do we want to add Microsoft as well?

Reply:
Yes, I mentioned that as well.

> Please add a Microsoft copyright to files with substantial new changes.

Reply:
Updated all the files to have a Microsoft copyright.